core/src/ast/mod.rs

```diff
@@ -203,6 +206,48 @@ impl ToSql for Statement {
             Statement::AlterTable { name, operation } => {
                 format!("ALTER TABLE {} {}", name.to_sql(), operation.to_sql())
             }
+            Statement::DropTable { if_exists, names } => {
+                let names = names
+                    .iter()
+                    .map(ToSql::to_sql)
+                    .collect::<Vec<_>>()
+                    .join(", ");
+                match if_exists {
+                    true => format!("DROP TABLE IF EXISTS {}", names),
+                    false => format!("DROP TABLE {}", names),
+                }
+            }
+            #[cfg(feature = "index")]
+            Statement::CreateIndex {
+                name,
+                table_name,
+                column: _column,
+            } => {
+                format!(
+                    "CREATE INDEX {} ON {} (..order_by_expr..)",
```

Comment:
[context]: <https://github.com/gluesql/gluesql/pull/812> must be completed to enter. TODO comments will be added.